raiden/tests/integration/test_version.py

```diff
@@ -0,0 +1,73 @@
+import pytest
+import tempfile
+import re
+import os
+import shutil
+
+from ethereum import _solidity
+
+from raiden.utils import get_contract_path
+from raiden.exceptions import ContractVersionMismatch
+
+
+def replace_contract_version(file_path, new_version):
+    original = open(file_path, 'r')
+    replaced = tempfile.NamedTemporaryFile()
+    for line in original.readlines():
+        if re.match(r'^\s*string constant public contract_version = "\d+.\d+.[\d+\_]";\s*$', line):
```

Comment:
This regex is duplicated and here is also needs fixing, please move it to a constant in a single place and import it here.